### ISSUE TITLE
Netbeans 8.1 starts but no UGS inside

### ISSUE THREAD

**Issue by MitchBradley** [winder/Universal-G-Code-Sender#665]
UGS Platform nightly build as of 6/2/2017.  Latest JRE 8 as of same day.  Win 7 Pro 64 bit.

Running ugsplatform64.exe brings up the NetBeans Platform 8.1 window but there is no UGS application inside - it is just a big gray window with a generic NetBeans menu bar.

![image](https://cloud.githubusercontent.com/assets/4861133/26749252/b8ceb9e6-47a2-11e7-8436-79f9d7a9f597.png)

I tried changing the heap size in ugsplatform.conf; same results with 128m, 256m, 512m.

I turned on console logging and ran from a Command window, giving:

```
C:\Users\wmb\Downloads\ugsplatform\ugsplatform\bin>ugsplatform64.exe

C:\Users\wmb\Downloads\ugsplatform\ugsplatform\bin>

The launcher has determined that the parent process has a console and will reuse it for its own console output.
Closing the console will result in termination of the running program.
Use '--console suppress' to suppress console output.
Use '--console new' to create a separate console window.
-------------------------------------------------------------------------------
>Log Session: Friday, June 2, 2017 2:47:49 PM HST
>System Info:
  Product Version         = NetBeans Platform 8.1 (Build 201510222201)
  Operating System        = Windows 7 version 6.1 running on amd64
  Java; VM; Vendor        = 1.8.0_131; Java HotSpot(TM) 64-Bit Server VM 25.131-b11; Oracle Corporation
  Runtime                 = Java(TM) SE Runtime Environment 1.8.0_131-b11
  Java Home               = C:\Program Files\Java\jdk1.8.0_131\jre
  System Locale; Encoding = en_US (serverlaunch); Cp1252
  Home Directory          = C:\Users\wmb
  Current Directory       = C:\Users\wmb\Downloads\ugsplatform\ugsplatform
  User Directory          = C:\Users\wmb\AppData\Roaming\.ugsplatform\dev
  Cache Directory         = C:\Users\wmb\AppData\Roaming\.ugsplatform\dev\var\cache
  Installation            = C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform
                            C:\Users\wmb\Downloads\ugsplatform\ugsplatform\ugsplatform
                            C:\Users\wmb\Downloads\ugsplatform\ugsplatform\java
                            C:\Users\wmb\Downloads\ugsplatform\ugsplatform\extra
                            C:\Users\wmb\Downloads\ugsplatform\ugsplatform\ide
                            C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform
  Boot & Ext. Classpath   = C:\Program Files\Java\jdk1.8.0_131\jre\lib\resources.jar;C:\Program Files\Java\jdk1.8.0_131\
jre\lib\rt.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\sunrsasign.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\jsse
.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\jce.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\charsets.jar;C:\Progr
am Files\Java\jdk1.8.0_131\jre\lib\jfr.jar;C:\Program Files\Java\jdk1.8.0_131\jre\classes;C:\Program Files\Java\jdk1.8.0
_131\jre\lib\ext\access-bridge-64.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\cldrdata.jar;C:\Program Files\Java\
jdk1.8.0_131\jre\lib\ext\dnsns.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\jaccess.jar;C:\Program Files\Java\jdk1
.8.0_131\jre\lib\ext\jfxrt.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\localedata.jar;C:\Program Files\Java\jdk1.
8.0_131\jre\lib\ext\nashorn.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\sunec.jar;C:\Program Files\Java\jdk1.8.0_
131\jre\lib\ext\sunjce_provider.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\sunmscapi.jar;C:\Program Files\Java\j
dk1.8.0_131\jre\lib\ext\sunpkcs11.jar;C:\Program Files\Java\jdk1.8.0_131\jre\lib\ext\zipfs.jar
  Application Classpath   = C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\boot.jar;C:\Users\wmb\Downloads\
ugsplatform\ugsplatform\platform\lib\org-openide-modules.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib
\org-openide-util-lookup.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\org-openide-util-ui.jar;C:\User
s\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\org-openide-util.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform
\platform\lib\locale\boot_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\boot_pt_BR.jar;C:\Us
ers\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\boot_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform
\platform\lib\locale\boot_zh_CN.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-modul
es_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-modules_pt_BR.jar;C:\Users\wmb\
Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-modules_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsp
latform\platform\lib\locale\org-openide-modules_zh_CN.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\lo
cale\org-openide-util-lookup_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util-
lookup_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util-lookup_ru.jar;C:\Us
ers\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util-lookup_zh_CN.jar;C:\Users\wmb\Downloads\u
gsplatform\ugsplatform\platform\lib\locale\org-openide-util-ui_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\pla
tform\lib\locale\org-openide-util-ui_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-op
enide-util-ui_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util-ui_zh_CN.jar;C:
\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util_ja.jar;C:\Users\wmb\Downloads\ugsplatf
orm\ugsplatform\platform\lib\locale\org-openide-util_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\l
ib\locale\org-openide-util_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\lib\locale\org-openide-util_zh
_CN.jar;C:\Program Files\Java\jdk1.8.0_131\lib\dt.jar;C:\Program Files\Java\jdk1.8.0_131\lib\tools.jar
  Startup Classpath       = C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\asm-all-5.0.1.jar;C:\Users\wmb\
Downloads\ugsplatform\ugsplatform\platform\core\core-base.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\co
re\core.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\org-netbeans-libs-asm.jar;C:\Users\wmb\Download
s\ugsplatform\ugsplatform\platform\core\org-openide-filesystems-compat8.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatfo
rm\platform\core\org-openide-filesystems.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\core-ba
se_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\core-base_pt_BR.jar;C:\Users\wmb\Downloads
\ugsplatform\ugsplatform\platform\core\locale\core-base_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\c
ore\locale\core-base_zh_CN.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\core_ja.jar;C:\Users\
wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\core_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform
\platform\core\locale\core_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\core_zh_CN.jar;C:\
Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-netbeans-libs-asm_ja.jar;C:\Users\wmb\Downloads\ugs
platform\ugsplatform\platform\core\locale\org-netbeans-libs-asm_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform
\platform\core\locale\org-netbeans-libs-asm_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\o
rg-netbeans-libs-asm_zh_CN.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-openide-filesyste
ms-compat8_ja.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-openide-filesystems-compat8_pt
_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-openide-filesystems-compat8_ru.jar;C:\Us
ers\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-openide-filesystems-compat8_zh_CN.jar;C:\Users\wmb\Do
wnloads\ugsplatform\ugsplatform\platform\core\locale\org-openide-filesystems_ja.jar;C:\Users\wmb\Downloads\ugsplatform\u
gsplatform\platform\core\locale\org-openide-filesystems_pt_BR.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platfor
m\core\locale\org-openide-filesystems_ru.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform\core\locale\org-ope
nide-filesystems_zh_CN.jar;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\ugsplatform\core\locale\core_ugsplatform.jar
-------------------------------------------------------------------------------
INFO [org.netbeans.core.startup.NbEvents]: Turning on modules:
        org.openide.util.lookup [8.32.1 201510222201]
        org.openide.util [9.5.1 201510222201]
        org.openide.util.ui [9.4.1 201510222201]
        org.openide.modules [7.47.1 201510222201]
        org.netbeans.api.annotations.common/1 [1.26.1 201510222201]
        org.openide.filesystems [9.8.1 201510222201]
        org.openide.awt [7.65.1 201510222201]
        org.netbeans.api.progress/1 [1.46.1 201510222201]
        org.netbeans.api.progress.nb [1.46.1 201510222201]
        org.openide.dialogs [7.41.1 201510222201]
        org.openide.nodes [7.42.1 201510222201]
        org.openide.windows [6.74.1 201510222201]
        org.netbeans.modules.editor.mimelookup/1 [1.39.1 201510222201]
        org.openide.text [6.66.1 201510222201]
        org.netbeans.swing.tabcontrol [1.54.1 201510222201]
        org.netbeans.swing.outline [1.32.1 201510222201]
        org.openide.explorer [6.61.1 201510222201]
        org.openide.actions [6.38.1 201510222201]
        org.netbeans.modules.queries/1 [1.42.1 201510222201]
        org.netbeans.api.templates [1.6.1 201510222201]
        org.openide.filesystems.nb [9.7.1 201510222201]
        org.openide.loaders [7.63.1 201510222201]
        org.netbeans.api.intent [1.1.1 201510222201]
        org.netbeans.api.io [1.2.1 201510222201]
        org.openide.io [1.48.1 201510222201]
        org.openide.filesystems.compat8 [9.7.1 201510222201]
        org.netbeans.swing.plaf [1.40.1 201510222201]
        org.netbeans.spi.quicksearch [1.26.1 201510222201]
        org.netbeans.modules.templates/1 [1.6.1 201510222201]
        org.netbeans.libs.asm [5.2.1 201510222201]
        org.netbeans.bootstrap/1 [2.74.1 201510222201]
        org.netbeans.core.startup.base [1.61.1.1 1 201510222201]
        org.netbeans.core.startup/1 [1.61.1.1 201510222201]
        org.netbeans.modules.settings/1 [1.48.1 201510222201]
        org.netbeans.modules.sendopts/2 [2.34.1 201510222201]
        org.netbeans.modules.sampler [1.13.1 201510222201]
        org.netbeans.modules.progress.ui [1.32.1 201510222201]
        org.netbeans.modules.print [7.25.1 201510222201]
        org.netbeans.modules.keyring [1.23.1 201510222201]
        org.netbeans.core/2 [3.49.1 201510222201]
        org.netbeans.modules.options.api/1 [1.44.1 201510222201]
        org.netbeans.modules.options.keymap [1.36.1 201510222201]
        org.netbeans.libs.jna/1 [1.41.1 201510222201]
        org.netbeans.modules.masterfs/2 [2.54.1.2 2 201510222201]
        org.netbeans.modules.masterfs.windows [1.15.1 201510222201]
        org.netbeans.modules.masterfs.ui [2.2.1.2 201510222201]
        org.netbeans.modules.masterfs.nio2 [1.14.1 201510222201]
        org.netbeans.modules.editor.util/1 [1.61.1 201510222201]
        org.netbeans.modules.lexer/2 [1.62.1.1 1 201510222201]
        org.netbeans.modules.keyring.fallback [1.7.1 201510222201]
        org.netbeans.modules.keyring.impl [1.23.1 201510222201]
        org.netbeans.modules.favorites/1 [1.45.1 201510222201]
        org.netbeans.modules.editor.settings.lib [1.50.1.1 1 201510222201]
        org.netbeans.modules.editor.settings/1 [1.55.1 201510222201]
        org.netbeans.modules.editor.settings.storage/1 [1.50.1.1 201510222201]
        org.netbeans.modules.editor.fold/1 [1.43.1 201510222201]
        org.netbeans.modules.editor.document [1.5.1.1 1 201510222201]
        org.netbeans.modules.editor.lib2/1 [2.4.1.44.1 44 201510222201]
        org.netbeans.modules.editor.indent/2 [1.42.1 201510222201]
        org.netbeans.modules.editor.guards/1 [1.31.1 201510222201]
        org.netbeans.modules.editor.lib/3 [4.3.1.22.1.44 22 201510222201]
        org.netbeans.modules.editor.plain.lib/1 [1.32.1 201510222201]
        org.netbeans.modules.editor.fold.nbui [1.11.1.44 201510222201]
        org.netbeans.modules.editor/3 [1.84.1.5.1.22.44 5 201510222201]
        org.netbeans.modules.editor.plain/2 [1.34.1 201510222201]
        org.netbeans.modules.editor.mimelookup.impl/1 [1.31.1 201510222201]
        org.netbeans.modules.editor.actions/1 [1.29.1.44 201510222201]
        org.netbeans.modules.autoupdate.services [1.51.1 201510222201]
        org.netbeans.modules.autoupdate.cli [1.12.1 201510222201]
        org.netbeans.core.windows/2 [2.80.1 201510222201]
        org.netbeans.core.ui/1 [1.44.1 201510222201]
        org.netbeans.libs.osgi [1.20.1 201510222201]
        org.netbeans.libs.felix [2.14.1 201510222201]
        org.netbeans.modules.autoupdate.ui [1.44.1 201510222201]
        org.netbeans.modules.core.kit [1.29.1 201510222201]
        org.netbeans.libs.jna.platform/1 [1.10.1 201510222201]
        org.netbeans.core.output2/1 [1.40.1 201510222201]
        org.netbeans.core.network [1.9.1 201510222201]
        org.netbeans.core.netigso [1.30.1 201510222201]
        org.netbeans.core.nativeaccess/1 [1.30.1 201510222201]
        org.netbeans.core.multitabs/1 [1.11.1.1 1 201510222201]
        org.netbeans.core.io.ui/1 [1.29.1 201510222201]
        com.willwinder.ugs.platform.branding [2.0 2.0-20170528 201705280409]
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy resolver: Windows
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy reloading succeeded.
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy - mode: direct
INFO [org.netbeans.core.network.proxy.NetworkProxyReloader]: System network proxy: fell to default (correct if direct mo
de went before)
INFO [org.netbeans.ui.metrics.laf]: USG_LOOK_AND_FEEL
Diagnostic information
Input arguments:
        -Xms512m
        -Dnetbeans.logger.console=true
        -ea
        -Xmx768m
        -XX:MaxPermSize=384m
        -Dnetbeans.user.dir=C:\Users\wmb\Downloads\ugsplatform\ugsplatform\bin
        -Djdk.home=C:\Program Files\Java\jdk1.8.0_131
        -splash:C:\Users\wmb\AppData\Roaming\.ugsplatform\dev\var\cache\splash.png
        -Dnetbeans.home=C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform
        -Dnetbeans.user=C:\Users\wmb\AppData\Roaming\.ugsplatform\dev
        -Dnetbeans.default_userdir_root=
        -XX:+HeapDumpOnOutOfMemoryError
        -XX:HeapDumpPath=C:\Users\wmb\AppData\Roaming\.ugsplatform\dev\var\log\heapdump.hprof
        -Dsun.awt.keepWorkingSetOnMinimize=true
        -Dnetbeans.dirs=C:\Users\wmb\Downloads\ugsplatform\ugsplatform\platform;C:\Users\wmb\Downloads\ugsplatform\ugspl
atform\ugsplatform;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\java;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\ex
tra;C:\Users\wmb\Downloads\ugsplatform\ugsplatform\ide
        exit
Compiler: HotSpot 64-Bit Tiered Compilers
Heap memory usage: initial 512.0MB maximum 683.0MB
Non heap memory usage: initial 2.4MB maximum -1b
Garbage collector: PS Scavenge (Collections=1 Total time spent=0s)
Garbage collector: PS MarkSweep (Collections=1 Total time spent=0s)
Classes: loaded=4390 total loaded=4390 unloaded 0
INFO [org.netbeans.core.ui.warmup.DiagnosticTask]: Total memory 8,471,691,264
INFO [null]: Total physical memory 8,471,691,264

```

**Comment by winder** [comment]
@MitchBradley very strange, I don't see any errors in the log. If you open up the program again and go to the `about` page you'll see a `User directory` and `Cache directory`. Try deleting those directories and reloading to see if it helps.

**Comment by MitchBradley** [comment]
Thanks, deleting the User directory (dev), which included the Cache
directory as a subdirectory (dev/var/cache), solved the problem.

I have an idea about how the problem may have come about.  The first
time I tried to run the program, I had JRE 8 installed, but there was
also an older JDK 7 installation on my computer.  I think that the
program may have chosen the old JDK instead of the newer JRE, and then
failed, possibly leaving bad data in the User/Cache directories.  The
reason I suspect this is because I saw a reference to jdk 1.7 in an
earlier copy of the log.
